3.0/vector_tile.proto

```diff
@@ -69,6 +112,12 @@ message Tile {
                 // See https://github.com/mapbox/vector-tile-spec/issues/47
                 optional uint32 extent = 5 [ default = 4096 ];
 
+                repeated string string_values = 7;
+                repeated double double_values = 8 [ packed = true ];
+                repeated float float_values = 9 [ packed = true ];
+                repeated sfixed64 sfixed64_values = 10 [ packed = true ];
+                repeated fixed64 fixed64_values = 11 [ packed = true ];
+
```

Comment:
I suggest these should get "logical" names like `signed_integer_values` or so instead of ones based on the encoding `sfixed...`.

Reply:
Also fine with me.